### DISCUSSION TITLE
Cataract Eyes in EEVEE

### DISCUSSION THREAD

**Discussion by NOSALIS** [soupday/cc3_blender_tools#16] [Q&A] [answered]
![Eyes-Blind](https://user-images.githubusercontent.com/89745990/131261667-3b5dbd7b-2d21-4793-8c4d-72cf211af80c.png)

I get that this may be tuned for Cycles but is there a quick way to make the irises render correctly?

**Comment by soupday** [comment, marked as answer]
You just need to turn on **Screen Space Reflections** and then under that, **Refraction** in the render properties.

**Reply by soupday** [reply]
Note: The add-on is meant to turn this on automatically, but if it was crashing because of the bug you encountered in your other report, then it would not have gotten to code that enables this.

**Reply by NOSALIS** [reply]
Makes sense. It works now! Bout to post a reply on my other issue with more info.
